webpack.config.js

```diff
@@ -86,6 +86,7 @@ module.exports = {
                 exclude: /(node_modules)/,
                 loader: 'svg-loader',
             },
+            ...require('@guardian/atom-renderer/webpack/css')(path.join(__dirname, 'static', 'src', 'stylesheets', 'atoms', 'vars.scss')),
```

Comment:
should the args here be an object, so we can extend it more easily? (also helps document why that's being sent)

Reply:
will do!